iot-hub/Samples/device/DeviceReconnectionSample/readme.md

```diff
@@ -0,0 +1,56 @@
+# Device Reconnection Sample
+
+This sample code demonstrates the various connection status changes and connection status change reasons the Device Client can return, and how to handle them.
+
+The device client exhibits the following connection status changes with reason:
```

Comment:
Essentially, connection status changes come down to a few actions, right?

1. Use the client
2. Wait to use the client
3. Reinitialize the client
4. Abandon the app - requires human intervention

Right?

Reply:
yes, correct

Reply:
I'd make that clear in this readme, and for each connection status reference this enum.